package.json

```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
-    "format:check": "npx prettier --check \"**/*.md\"",
-    "format:write": "prettier --write"
+    "format:check": "npx prettier --check '**/*.md'",
+    "format:write": "prettier --write '**/*.md'"
```

Comment:
Il `format:write` serve anche a [file lefthook](https://github.com/Il-Libro-Open-Source/book/blob/c860dd49ce5a950f463c50faaa20b527ffd10c92/lefthook.yml#L6) che prende in input i file in staged, per questo motivo non ci sono file, directory o glob in input.
In [CONTRIBUTING.md](https://github.com/Il-Libro-Open-Source/book/blob/c860dd49ce5a950f463c50faaa20b527ffd10c92/CONTRIBUTING.md?plain=1#L20) abbiamo detto cosa fare per formattare il codice, probabilmente è meglio separare i 2 script per evitare incomprensioni, una cosa del tipo:
- `format:write:automation: prettier --write`
- `format:write: prettier --write '**/*.md'`

Che ne pensi?

Reply:
Direi perfetto se li smistiamo IMO

Reply:
Si, concordo con il separare gli script: il "serve anche" non è una buona pratica, se ci sono 2 scopi/domini vanno fatti 2 script, da qui l'incomprensione

Reply:
Splittati gli script, grazie